cpu/rpx0xx/Makefile.include

```diff
@@ -0,0 +1,19 @@
+PICO_DEBUG      ?=
+ROM_LEN         ?= 2097152 # = 2 MiB used in the RPi Pico
```

Comment:
since the MCU doesn't come with flash, shouldn't this always be set by the board?

Reply:
It is likely that most boards just use the reference flash chip, as the 2nd stage bootloader will then just work. Not every flash ship supports XIP in the same way, requiring other flash chips to use a different 2nd stage bootloader.

The board can still overwrite this value, if it indeed used a different value. IMO having the "reference flash chip" as default is sensible.